### PR TITLE
Dashboard working with documentation

### PR DESCRIPTION
**Pull request by luoyuctl**
What this PR does / why we need it:
Help new users to quickly get started with the dashboard system

Special notes for your reviewer:
thx

Specified Reivewers:
Kindhearted person

### CODE REVIEW

File: docs/3.21/manual/monitor/dashboard/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Dashboard 
+
+## Quick start 
+<br>微服务治理->运维大盘->新建运维大盘->编辑模式->新增图表->选择图表类型--柱状图
+
+![](http://terminus-paas.oss-cn-hangzhou.aliyuncs.com/paas-doc/2021/04/14/5c4db078-cd81-4543-8d26-c499a684b3f9.png)
```
Comment (daskyrk) [inline docs/3.21/manual/monitor/dashboard/dashboard.md:6]: should add image alt info

**Review by daskyrk — COMMENTED** [review]
Add a sidebar menu item in `docs/.vuepress/sidebar/zh.js`

File: docs/3.21/manual/monitor/dashboard/dashboard.md (hunk 1 above)
Comment (luoyuctl) [reply docs/3.21/manual/monitor/dashboard/dashboard.md:6]: Is that too junior to user?

File: docs/3.21/manual/monitor/dashboard/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Dashboard 
+
+## Quick start 
+<br>微服务治理->运维大盘->新建运维大盘->编辑模式->新增图表->选择图表类型--柱状图
```
Comment (daskyrk) [inline docs/3.21/manual/monitor/dashboard/dashboard.md:4]: Show the operation path as follows: 
> DevOps 平台 -> 我的项目 -> 待办事项

File: docs/3.21/manual/monitor/dashboard/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+![](http://terminus-paas.oss-cn-hangzhou.aliyuncs.com/paas-doc/2021/04/14/5c4db078-cd81-4543-8d26-c499a684b3f9.png)
+
+### <br>进入图表编辑器页面
```
Comment (daskyrk) [inline docs/3.21/manual/monitor/dashboard/dashboard.md:8]: `<br>` is useless

**Comment by luoyuctl** [comment]
> Add a sidebar menu item in `docs/.vuepress/sidebar/zh.js`

done

File: docs/3.21/manual/monitor/dashboard/dashboard.md (hunk 2 above)
Comment (luoyuctl) [reply docs/3.21/manual/monitor/dashboard/dashboard.md]: done

File: docs/3.21/manual/monitor/dashboard/dashboard.md (hunk 3 above)
Comment (luoyuctl) [reply docs/3.21/manual/monitor/dashboard/dashboard.md]: Already done

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**